levenshtein/racket/run.rkt

```diff
@@ -1,18 +1,18 @@
 #lang racket
-(require (file "../../lib/racket/benchmark.rkt")
-         "levenshtein.rkt"
-         racket/cmdline
-         racket/file)
+
+(require "../../lib/racket/benchmark.rkt"
+         "levenshtein.rkt")
 
 (command-line
-  #:args args
-  (define run-ms (string->number (list-ref args 0)))
-  (define warmup-ms (string->number (list-ref args 1)))
-  (define words (file->lines (list-ref args 2)))
+ #:args (run-ms-str warmup-ms-str words-str)
+ (define run-ms (string->number run-ms-str))
+ (define warmup-ms (string->number warmup-ms-str))
+ (define words (string->number words-str))
```

Comment:
The third arg is a filename. We still need to do something like:

```suggestion
 (define words (file->lines words-str))
```

Reply:
Yes, this change was too automatic and I didn't review. Will fix.

Reply:
Fixed in 8d67795 (racket: simplify and format argument parsing, 2025-02-09).